funlet-whisper/.env

```diff
@@ -1,3 +1,7 @@
+# description: If "true", check that the recipient is human by having them press a button.
+# format: text
+FUNLET_WHISPER_HUMAN_CHECK=false
```

Comment:
interesting, we should consider adding a checkbox field to CE sometime

Reply:
Agreed! I was surprised it isn't in there yet.